.azure-devops/templates/trigger-tests.yml

```diff
@@ -33,64 +30,82 @@ jobs:
 - job: 'testCentral'
   displayName: 'Test IoT Central'
   condition: and(succeeded(), eq('${{ parameters.testCentral }}', true))
+  strategy:
+    matrix: $[ variables['pythonVersions'] ]
+    maxParallel: 2
```

Comment:
Any technology areas that do not provision the target resource for test must have a maxParallel of 1.

Reply:
Done